test/core/bad_ssl/generate_tests.bzl

```diff
@@ -42,7 +42,10 @@ def grpc_bad_ssl_tests():
         grpc_cc_binary(
             name = "bad_ssl_%s_server" % t,
             srcs = ["servers/%s.cc" % t],
-            deps = [":bad_ssl_test_server"],
+            deps = [
+                ":bad_ssl_test_server",
+                "//test/core/util:grpc_test_util",
```

Comment:
Why is this dependency needed?

Reply:
it's not needed, good catch. This carried over from a previous incarnation.